source/directives/uiTreeNode.js

```diff
@@ -116,10 +117,12 @@
               tagName = scope.$element.prop('tagName');
 
               if (tagName.toLowerCase() === 'tr') {
-                placeElm = angular.element($window.document.createElement(tagName));
-                tdElm = angular.element($window.document.createElement('td'))
+                placeElm = angular.element($window.document.createElement(tagName))
                   .addClass(config.placeholderClass);
-                placeElm.append(tdElm);
+                for (i = 0; i < scope.$element[0].children.length; i++) {
```

Comment:
Would this also work if we replace the for-loop with setting the `colspan` attribute of the `tr`? Eg. `placeElm.attr('colspan', scope.$element[0].children.length)`.